sys-auth/libfprint/libfprint-1.94.9-r1.ebuild

```diff
@@ -69,6 +69,10 @@ src_unpack() {
 }
 
 src_configure() {
+	if use tod && tc-ld-is-lld; then
+		eapply "${FILESDIR}/${PN}-1.94.9-tod-fix-linking-with-lld.patch"
+	fi
```

Comment:
Can we apply this conditional to only `tod`? Or does this then break other linkers?

Reply:
I tested non-tod version with LLD and it didn't fail linking. So I guess this applies to tod only.

Reply:
If we're going to do this, it is far better to do `append-ldflags $(test-flags-CCLD -Wl,--undefined-version)` as a workaround instead until upstream have analysed it. Please see my other review comment.

Reply:
Thank you sam_, for now I removed the patch and have added this as a workaround till we have something from upstream. Till now unfortunately there's no activity on the upstream issue.